packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -0,0 +1,90 @@
+---
+id: Upgrade
+title: Major release upgrade guide
+section: get-started
+source: Major-release-upgrade
+---
+
+# PatternFly 5 
+
+Our latest major release introduces new support and functionality to PatternFly, including: 
+
+- React 18 support, 
+- Official dark theme support, 
+- An enhanced system of CSS classes and variables, 
+- And more!
+
+To learn about the most significant changes in this release, read our [release highlights](/get-started/release-higlights.md). A detailed list of all changes can be found in our [major release notes](get-started/upgrade.md#major-release-notes). 
+
+This guide outlines the major steps you should take to upgrade your product's codebase from PatternFly 4 to PatternFly 5. 
+
+## Get help 
+
+If you need support as you upgrade to PatternFly 5, the PatternFly team is here to help. Reach out to us on [Slack](https://join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ) or ask a question on our [GitHub discussion board](https://github.com/orgs/patternfly/discussions). We'll always do our best to answer your questions and connect you with the right people quickly. 
+
+## Upgrade your product's codebase
+
+When you upgrade your product to PatternFly 5, several breaking changes will likely be introduced to your product’s codebase. We are using a suite of Codemods to simplify and streamline the upgrade process. Instead of requiring you to manually identify all errors and issues in your codebase, you can run our Codemods to quickly identify and fix major issues. Keep in mind that some changes will still require manual intervention, but our Codemods can automatically fix a large amount of issues and flag any issues that do require manual work.
+
+To utilize our Codemods, refer to the following instructions. You can also [view the project on GitHub](https://github.com/patternfly/pf-codemods/) for additional details.
+
+###  Run our Codemods
+
+**Note:** Running Codemods after making manual changes will lead to even more manual intervention and cleanup. Because of this, we strongly recommend running Codemods _before_ making any manual changes to your codebase.
+
+To run our Codemods, complete the following steps:
+
+1. Run the following command, adding in the path to your product's source code: 
+
+```{
+npx @patternfly/pf-codemods@latest <path to your source code> --exclude menu-search-rename,alert-remove-titleHeadingLevel,datetimepicker-warn-helperText,formgroup-remove-helpertextProps,formhelpertext-remove-props,alert-remove-titleHeadingLevel
+}
+```
```

Comment:
If you add 4 spaces before the backticks before all of the lines, it will indent that block like this. Also I think you have an unnecessary `}` at the end of the block. I think the one at the beginning is needed to get the codeblock to render (that's how it works in the core workspace anyways), but the one at the end shouldn't be necessary./

<img width="856" alt="Screenshot 2023-06-21 at 9 37 08 PM" src="https://github.com/patternfly/patternfly-org/assets/35148959/05c42b96-9a97-4354-a920-33b56968b52c">


```suggestion
    ```{
    npx @patternfly/pf-codemods@latest <path to your source code> --exclude menu-search-rename,alert-remove-titleHeadingLevel,datetimepicker-warn-helperText,formgroup-remove-helpertextProps,formhelpertext-remove-props,alert-remove-titleHeadingLevel
    ```
```